src/tests/integration_tests/test_qa_agent_with_vectorSharedMemory.py

```diff
@@ -0,0 +1,80 @@
+Australia is wider than the moon. The moon sits at 3400km in diameter, while Australia’s diameter from east to west is almost 4000km.
+'Mellifluous' is a sound that is pleasingly smooth and musical to hear.
+The Spice Girls were originally a band called Touch. "When we first started [with the name Touch], we were pretty bland," Mel C told The Guardian in 2018. "We felt like we had to fit into a mould."
+Emma Bunton auditioned for the role of Bianca Butcher in Eastenders. Baby Spice already had a small part in the soap back in the 90s but tried out for a full-time role. She was pipped to the post by Patsy Palmer but ended up auditioning for the Spice Girls not long after.
+Human teeth are the only part of the body that cannot heal themselves. Teeth are coated in enamel which is not a living tissue.
+It's illegal to own just one guinea pig in Switzerland. It's considered animal abuse because they're social beings and get lonely.
+The Ancient Romans used to drop a piece of toast into their wine for good health - hence why we 'raise a toast'.
+The heart of a shrimp is located in its head. They also have an open circulatory system, which means they have no arteries and their organs float directly in blood.
+Amy Poehler was only seven years older than Rachel McAdams when she took on the role of "cool mom" in Mean Girls. Rachel was 25 as Regina George - Amy was 32 as her mum.
+People are more creative in the shower. When we take a warm shower, we experience an increased dopamine flow that makes us more creative.
+Baby rabbits are called kits. Cute!
+my dog died on march 2021.
+The unicorn is the national animal of Scotland. It was apparently chosen because of its connection with dominance and chivalry as well as purity and innocence in Celtic mythology.
+The first aeroplane flew on December 17,1903 and  . Wilbur and Orville Wright made four brief flights at Kitty Hawk, North Carolina, with their first powered aircraft, aka the first airplane.
+Venus is the only planet to spin clockwise. It travels around the sun once every 225 Earth days but it rotates clockwise once every 243 days.
+Nutmeg is a hallucinogen. The spice contains myristicin, a natural compound that has mind-altering effects if ingested in large doses.
+A 73-year-old bottle of French Burgundy became the most expensive bottle of wine ever sold at auction in 2018, going for $558,000 (approx £439,300). The bottle of 1945 Romanee-Conti sold at Sotheby for more than 17 times its original estimate of $32,000."""
+session_id = "6"
+meta_data = {
+    "session_id": f"{session_id}",
+    "file_name": "rtgfqq",
+    "file_type": "pdf",
+    "title": "NoMeaning",
+    "data_type": "user_input",
+}
+
+
+@pytest.fixture
+def config_logger():
+    logger.remove()
+    logger.add(sys.stderr, level="DEBUG")
+
+
+@pytest.mark.external_api
+def test_shared_memory_with_vector(config_logger, get_llm):
+    llm = ChatOpenAI(model_name="gpt-3.5-turbo")
```

Comment:
Please make this test work offline. See `test_ml_engineer_succeeds` and the accompanying JSON file of cached test data for an example.